lambdas/get-addresses/src/lib/param-store-client.ts

```diff
@@ -0,0 +1,3 @@
+import { SSMClient } from "@aws-sdk/client-ssm";
+
+export const SsmClient = new SSMClient({ region: "eu-west-2" });
```

Comment:
Should we be explicitly setting the region here, can we maybe use AWS_REGION environment variable see: https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-region.html

Reply:
we could use an environment variable to set the region, but we can only run out of eu-west-2 so not sure what setting it via an env would give us